.zuul.d/config.yaml

```diff
@@ -0,0 +1,51 @@
+- project:
+    name: SovereignCloudStack/standards
+    default-branch: main
+    merge-mode: "squash-merge"
+    periodic-daily:
+      jobs:
+        - scs-check-pco-prod1
+        - scs-check-pco-prod2
+        - scs-check-pco-prod3
+        - scs-check-regio-a
+        - scs-check-wavestack
+    periodic-hourly:
+      jobs:
+        - scs-check-gx-scs
+    check:
+      jobs:
+        - scs-check-gx-scs
```

Comment:
```suggestion
```

You can run a job with a secret only in the pipeline which has set `post-review: true`, see https://zuul-ci.org/docs/zuul/latest/config/pipeline.html#attr-pipeline.post-review